packages/react-core/src/components/LoginPage/LoginPage.tsx

```diff
@@ -35,6 +35,8 @@ export interface LoginPageProps extends React.HTMLProps<HTMLDivElement> {
   loginTitle: string;
   /** Subtitle for the Login Main Body Header of the LoginPage */
   loginSubtitle?: string;
+  /** Header utilities for the Login Main Body Header of the LoginPage */
```

Comment:
nit - we might consider updating these descriptions to use sentence case?

Reply:
should I update this for all of the descriptions?

Reply:
yes - even our component names are lower cased when used in a sentence